templates/docker-compose.yml

```diff
@@ -37,7 +41,6 @@ services:
     hostname: lemmy
     networks:
       - lemmyinternal
-      - lemmyexternalproxy
```

Comment:
Is it Lemmy or Lemmy UI that sends notifications to other federated instances? Wouldn't we need the "external" network to send federated items to them? (posts/comments/etc)

Reply:
From my testing just now, it's Lemmy. I'll update the PR and add the "internal" flag back to the "lemmyinternal" network and put the "lemmy" container in both.